core/web/apiv2/auth.py

```diff
@@ -93,11 +84,25 @@ async def get_current_user(
     user = UserSensitive.find(username=username)
     if user is None:
         raise credentials_exception
-    if not user.enabled:
-        raise disabled_exception
+    request.state.username = user.username
     return user
 
 
+async def get_current_active_user(
+    current_user: Annotated[User, Security(get_current_user)]
```

Comment:
If memory serves, you could also do

```python
current_user: User = Security(get_current_user)
```

Which is the syntax we use above (e.g. for `cookie`)

Reply:
Fixed in 94d3b77797c246f61902aee4acab9285eea74f72